FileMapper/pickerPrompt_script.html

```diff
@@ -0,0 +1,215 @@
+<!--
```

Comment:
let's be consistent with naming of files, pickerPrompt_script.html -> picker_prompt_script.html

Also for all other files

Reply:
Yeah, i will name all the files in snake case.